### PR TITLE
trying to add riot chat badge

### PR DESCRIPTION
**Pull request by hrishi32**
Issue #76

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -16,6 +16,9 @@
    :alt: Join the chat at https://gitter.im/EinsteinPy-Project/EinsteinPy
    :target: https://gitter.im/EinsteinPy-Project/EinsteinPy?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
```
Comment (hrishi32) [inline README.rst]: added badge

File: README.rst [hunk 2]
```diff
@@ -16,6 +16,9 @@
...
 
+.. |riotchat| image:: https://img.shields.io/matrix/:einsteinpy:matrix.org.svg?style=flat-square
```
Comment (hrishi32) [inline README.rst]: added badge

File: README.rst [hunk 3]
```diff
@@ -16,6 +16,9 @@
    :alt: Join the chat at https://gitter.im/EinsteinPy-Project/EinsteinPy
    :target: https://gitter.im/EinsteinPy-Project/EinsteinPy?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
 
+.. |riotchat| image:: https://img.shields.io/matrix/einsteinpy:matrix.org.svg?style=flat-square
+   :target: https://riot.im/app/#/room/#einsteinpy:matrix.org
```
Comment (shreyasbapat) [inline README.rst]: Thanks a lot for this!

**Comment by shreyasbapat** [comment]
Hi @hrishi32  ,
I mean to say this file: https://github.com/einsteinpy/einsteinpy/blob/master/docs/source/index.rst

**Comment by shreyasbapat** [comment]
What I meant to say is: Badges in README and index should exactly match

**Comment by hrishi32** [comment]
> What I meant to say is: Badges in README and index should exactly match

Done :)

**Comment by shreyasbapat** [comment]
LGTM! 
Thanks for this extra effort! :rocket: 
Merging!

**Pull request merged**